lib/core/templates.html

```diff
@@ -573,5 +573,50 @@ <h3 class="RESHoverTitle" data-hover-element="0"></h3>
 				opacity: 1.0;
 			}
 		</style>
+
+		<div id="saveComments">
+			<div id="res-saveComments" class="linklisting">
+				<div class="res-module-description md">
+					<h1>Saving comments with RES</h1>
+					{{{moduleDescription}}}</div>
+				<div class="res-module-tip infobar">
+					<p><i>Keyboard Shortcuts</i> <a class="gearIcon" href="{{keyNavHash}}" title="RES settings"></a></p>
+					<ul>
+						<li><b>{{savePostKey}}</b>: save a submission.</li>
+						<li><b>{{saveCommentKey}}</b>: save a comment (to your reddit account).</li>
+						<li><b>{{saveRESKey}}</b>: save a comment with RES.</li>
+					</ul>
+				</div>
+				<div class="res-saveComments-list"></div>
+			</div>
+		</div>
+
+		<div id="saveComments-comment">
```

Comment:
Would be cool if this was all in the same template, in a `{{#comments}}` list.
Then you could eliminate `$wrapper.find('.res-module-description').hide();`, and wrap the description in `{{^comments}}{{/comments}}`, which will only be displayed if `comments` is falsy or an empty list.